sql/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -495,7 +495,7 @@ public void testTableConstraintsWithOrderBy() throws Exception {
     @Test
     public void testDefaultColumns() throws Exception {
         execute("select * from information_schema.columns order by table_schema, table_name");
-        assertEquals(464, response.rowCount());
+        assertEquals(465, response.rowCount());
```

Comment:
@mfussenegger Why/How is this affected?

Reply:
Because there is now an additional column. `information_schema.columns` also lists columns of system tables.